boards/FSM/Src/App/App_Brake.c

```diff
@@ -13,38 +14,44 @@
 #define BRAKE_PRESSURE_OC_THRESHOLD \
     (MAX_BRAKE_PRESSURE_PSI * BRAKE_PRESSURE_OC_THRESHOLD_V / BRAKE_PRESSURE_SENSOR_MAX_V)
 
+STATIC_DEFINE_APP_SET_PERIODIC_CAN_SIGNALS_IN_RANGE_CHECK(FsmCanTxInterface)
+
 struct Brake
 {
     struct InRangeCheck *pressure_in_range_check;
-
-    float min_pressure_psi;
-    float max_pressure_psi;
-
-    float (*get_pressure_psi)(void);
-    bool (*is_pressure_sensor_open_or_short_circuit)(void);
+    float (*get_primary_pressure_psi)(void);
```

Comment:
Lets change to front and back pressure sensor throughout please

Reply:
Fixed with d6cf3333dfa52536e7aa9ba469d1bec0af049787